test/common/upstream/health_checker_impl_test.cc

```diff
@@ -3639,34 +3548,35 @@ class GrpcHealthCheckerImplTestBase {
     std::vector<std::pair<std::string, std::string>> trailers;
   };
 
-  GrpcHealthCheckerImplTestBase()
-      : cluster_(new NiceMock<MockClusterMockPrioritySet>()),
-        event_logger_(new MockHealthCheckEventLogger()) {
+  GrpcHealthCheckerImplTestBase() : HealthCheckerTestBase() {
```

Comment:
you can (and must) omit the 0-arg construction of the base class. this is your clang-tidy error.